tasks/extensions/postgis.yml

```diff
@@ -7,21 +7,19 @@
 
 - name: PostgreSQL | Extensions | Make sure the postgis extensions are installed | Debian
   apt:
-    name: "{{item}}"
+    name: "{{ postgresql_ext_postgis_deps }}"
```

Comment:
@stejoo , when doing a single item, should it still follow the format of putting it onto a second line with a dash?

```
name:
  - "{{ postgresql_ext_postgis_deps }}"
```

That way, it's consistent with the other uses, and makes it easy to expand when a second item is required in the list.

Reply:
Sorry for never getting back to you on this. To be honest it really doesn't matter if it's a list with a single item or not. It works all the same, just looks odd in my opinion ;)
Expanding `postgresql_ext_postgis_deps` would work to add more packages, as would making a list out of it here and adding a second item.